<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Codigos Bidimensionales</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icono.ico</normaloff>icono.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imagenCodigo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>256</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>256</width>
      <height>256</height>
     </size>
    </property>
    <property name="text">
     <string>QR</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TextoCodigo">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerarCodigo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Codigo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generación del código</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Introduzca el mensaje que quiera codificar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="CargarCodigo"/>
    <addaction name="GuardarCodigo"/>
   </widget>
   <widget class="QMenu" name="menuPersonalizar">
    <property name="title">
     <string>Personalizar</string>
    </property>
    <widget class="QMenu" name="ColorFondo">
     <property name="title">
      <string>Color Fondo</string>
     </property>
    </widget>
    <widget class="QMenu" name="ColorCodigo">
     <property name="title">
      <string>Color Codigo</string>
     </property>
    </widget>
    <addaction name="ColorFondo"/>
    <addaction name="ColorCodigo"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuPersonalizar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="CargarCodigo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cargar Codigo</string>
   </property>
  </action>
  <action name="Guardar_Gris">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Gris</string>
   </property>
  </action>
  <action name="Guardar_Hilbert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hilbert</string>
   </property>
  </action>
  <action name="Guardar_Linear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="GuardarCodigo">
   <property name="text">
    <string>Guardar Codigo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
